### ISSUE TITLE
Make Bioplastic meltable in smeltery?

### ISSUE THREAD

**Issue by squeek502** [squeek502/VeganOption#26]
From kane_thornwyrd (Stranded Strained modpack dev) [here](http://minecraft.curseforge.com/modpacks/233775-stranded-strained/issues/1412):

> I even sometimes add new ways to get some stuffs, and in that context the possibility to smelt bio-plastic and cast it into other things.

Seems like it wouldn't hurt. Could even add potato starch -> melted bioplastic smelting recipe.

**Comment by jena-th** [comment]
:smiley: I just hope it won't be too much of an hassle to implement it, for the time being, people interested in that kind of possibilities can use MineTweaker and ModTweaker to make Vegan Option, Tinker's Construct and MFR work together: 

https://gist.github.com/kane-thornwyrd/a9e691a3857d73697269

Except that there is a weird bug with ModTweaker and TConstruct, the table recipes are well registered, but they just don't work, so, you may add something like this to be able to use the liquid plastic :

```
mods.tconstruct.Casting.addBasinRecipe(<MineFactoryReloaded:plastic>, <liquid:plastic> * 576, null, false, 20);
```

N.B. Plastic parts can still be made using the plastic made by smelting the starch in a furnace.

Note, that for now I've set the required temperature high to nerf the recipe a bit, ideally I would like a liquid starch registered, and use MineChem's Sodium Hydroxide and Hydrogen Chloride to register a new "alloy recipe" for liquid plastic and lower the required heat and materials for the whole process.

So basically, plastic made out of salt water, starch and energy :smile: .
At least an alternative to the rubber tree way.
